src/generator/src/cmd/generate.ts

```diff
@@ -114,7 +114,7 @@ async function generateAutorestConfig(logger: ILogger, readmePath: string, bicep
   // We expect a path format convention of <provider>/(preview|stable)/<yyyy>-<mm>-<dd>(|-preview)/<filename>.json
   // This information is used to generate individual tags in the generated autorest configuration
   // eslint-disable-next-line no-useless-escape
-  const pathRegex = /^(\$\(this-folder\)\/|)([^\/]+)\/[^\/]+\/(\d{4}-\d{2}-\d{2}(|-preview))\/.*\.json$/i;
+  const pathRegex = /^(\$\(this-folder\)\/|)([^\/]+)(?:\/[^\/]+)+\/(\d{4}-\d{2}-\d{2}(|-preview))\/.*\.json$/i;
```

Comment:
1. Does this change the match group number used below?
2. Please could you also update the comment above regarding path format convention?

Were you able to validate this change generates the correct output when the pipeline is executed?

Reply:
The match group numbers remain the same (`(?:...)` is a non-matching group).

The updated pipeline was executed on Friday and produced [this PR](https://github.com/Azure/bicep-types-az/pull/1082), which restored all the `Microsoft.Compute` types and had no other impact.